app/src/pages/playground/playgroundUtils.ts

```diff
@@ -164,12 +165,13 @@ function getModelConfigFromAttributes(
 ):
   | { modelConfig: ModelConfig; parsingErrors: never[] }
   | { modelConfig: null; parsingErrors: string[] } {
-  const { success, data } = modelNameSchema.safeParse(parsedAttributes);
+  const { success, data } = modelConfigSchema.safeParse(parsedAttributes);
   if (success) {
     return {
       modelConfig: {
         modelName: data.llm.model_name,
         provider: getModelProviderFromModelName(data.llm.model_name),
+        invocationParameters: data.llm.invocation_parameters,
```

Comment:
the error message below probably needs to be updated since it's only model name.